lib/rules/selector-max-class/README.md

```diff
@@ -0,0 +1,51 @@
+# selector-max-class
+
+Limit the number of classes in a [complex selector](https://www.w3.org/TR/selectors4/#complex).
```

Comment:
Very minor nitpick - let's change this to match the one in the rules list i.e.

"Limit the number of classes in a selector."